pkg/apis/kops/v1alpha3/cluster.go

```diff
@@ -240,6 +240,9 @@ type GCESpec struct {
 	NodeInstancePrefix *string `json:"nodeInstancePrefix,omitempty"`
 	// PDCSIDriver is the config for the PD CSI driver.
 	PDCSIDriver *PDCSIDriver `json:"pdCSIDriver,omitempty"`
+
+	// BinariesLocation is the location of the GCE cloud provider binaries.
+	BinariesLocation *string `json:"binariesLocation,omitempty"`
```

Comment:
This should go to `pkg/apis/kops/v1alpha2/cluster.go` too.

Reply:
@upodroid ^^^

Reply:
Where?  v1alphav2 doesn't implement cloudproviderspec

```
	CloudProvider kops.CloudProviderSpec `json:"-"`
```